Assets/MRTK/Tests/PlayModeTests/ScrollViewTests.cs

```diff
@@ -937,46 +1085,326 @@ public IEnumerator ContentClickHappensOnTouchUp()
+
+            // Completely visible items should have renderers enabled. Colliders should remain enabled for interaction.
+            Assert.IsTrue(clippedRenderers.Contains(renderer0), "Renderer 0 is not being clipped");
+            Assert.IsTrue(renderer0.enabled, "Renderer 0 is disabled");
+            Assert.IsTrue(collider0.enabled, "Collider 0 is disabled");
+
+            // Barelly visible content should still have renderers enabled. Colliders should be disabled for interaction.
+            Assert.IsTrue(clippedRenderers.Contains(renderer1), "Renderer 1 is not being clipped");
+            Assert.IsTrue(renderer1.enabled, "Renderer 1 is disabled");
+            Assert.IsFalse(collider1.enabled, "Collider 1 is enabled");
+
+            // Hidden content should have renderers disabled. Colliders should be disabled for interaction.
+            Assert.IsTrue(clippedRenderers.Contains(renderer2), "Renderer 2 is being clipped");
+            Assert.IsFalse(renderer2.enabled, "Renderer 2 is enabled");
+            Assert.IsFalse(collider2.enabled, "Collider 2 is enabled");
+
+            // Scrolling half item up
+            yield return hand.MoveTo(preTouchPos);
+            yield return hand.MoveTo(pastTouchPos);
+            yield return hand.MoveTo(scrollEngagedHalfPageUpPos);
+            yield return hand.MoveTo(initialPos);
+
+            clippedRenderers = scrollView.ClipBox.GetRenderersCopy().ToList();
+
+            // Partially visible content should have renderers enabled. Colliders should be enabled for interaction.
+            Assert.IsTrue(clippedRenderers.Contains(renderer0), "Renderer 0 is not being clipped");
+            Assert.IsTrue(renderer0.enabled, "Renderer 0 is disabled");
+            Assert.IsTrue(collider0.enabled, "Collider 0 is disabled");
+
+            Assert.IsTrue(clippedRenderers.Contains(renderer1), "Renderer 1 is not being clipped");
+            Assert.IsTrue(renderer1.enabled, "Renderer 1 is disabled");
+            Assert.IsTrue(collider1.enabled, "Collider 1 is disabled");
+
+            // Hidden content should have renderers disabled. Colliders should be disabled for interaction.
+            Assert.IsTrue(clippedRenderers.Contains(renderer2), "Renderer 2 is not being clipped");
+            Assert.IsFalse(renderer2.enabled, "Renderer 2 is enabled");
+            Assert.IsFalse(collider2.enabled, "Collider 2 is enabled");
+
+            // Removing content from scroll content should also remove its renderers from the scroll clipping box.
+            scrollView.RemoveItem(contentItems[0]);
+            clippedRenderers = scrollView.ClipBox.GetRenderersCopy().ToList();
+
+            // Object is still visible, but renderer should not be clipped.
+            Assert.IsFalse(clippedRenderers.Contains(renderer0), "Renderer 0 is being clipped");
+        }
+
+        /// <summary>
+        /// Tests if component can scroll content made of objects layouted manually without a BaseObjectCollection.
```

Comment:
> Tests if component can scroll content made of objects layouted manually without a BaseObjectCollection. [](start = 11, length = 104)

<3!!!

Reply:
codeflow destroyed my heart - i was just thinking when having a look at the tests if i have to have the grid object collection now all the time - and then that test magically appeared answering all my questions :D 
❤️